chart/genesis/values.yaml

```diff
@@ -0,0 +1,61 @@
+# Default values for genesis.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+secretMounts:
+  - name: 'docker-cert'
+    secretName: 'genesis-docker-cert'
+    mountPath: '/var/secrets'
+    readOnly: true
+
+replicaCount: 1
+
+image:
+  repository: gcr.io/whiteblock/genesis
+  tag: latest
+  pullPolicy: IfNotPresent
+
+nameOverride: ""
+fullnameOverride: ""
+
+service:
+  name: genesis
+  type: ClusterIP
+  port: 8000
+  protocol: TCP
+  targetPort: 8000
+
+environment:
+  - name: QUEUE_PASSWORD
+    valueFrom:
+      secretKeyRef:
+        name: genesis-platform-secret
+        key: queue_password
+  - name: QUEUE_USER
+    valueFrom:
+      secretKeyRef:
+        name: genesis-platform-secret
+        key: queue_user
+
+envFrom:
```

Comment:
does this conflict with the environment entries above?

Reply:
Looks like not.

Reply:
yeah these should be different environment variables